include/SDL3/SDL_process.h

```diff
@@ -0,0 +1,157 @@
+ * \param flags The flags to control the process options.
+ *
+ * \returns The newly created, now running process.
+ *
+ * \since This function is available since SDL 3.0.0.
+ */
+extern SDL_DECLSPEC SDL_Process *SDLCALL SDL_CreateProcess(const char * const *args, const char * const *env, SDL_ProcessFlags flags);
+
+/**
+ * Write to a process' stdin.
+ *
+ * The process must have been created with the SDL_EXEC_STDIN flag, else this function will fail.
+ *
+ * \param process The process to send data to.
+ * \param buffer A buffer containing the data to send to the process.
+ * \param size The amount of data to send to the process.
+ *
+ * \returns The number of bytes written, or -1 on error; call SDL_GetError() for more information.
+ *
+ * \since This function is available since SDL 3.0.0.
+ */
+extern SDL_DECLSPEC int SDLCALL SDL_WriteProcess(SDL_Process *process, const void *buffer, int size);
+
+/**
+ * Read from a process' stdout.
+ *
+ * The process must have been created with the SDL_EXEC_STDOUT flag, else this function will fail.
+ *
+ * \param process The process to read data from.
+ * \param buffer A buffer that will contain the data read from the process.
+ * \param size The amount of data to read from the process.
+ *
+ * \returns The number of bytes read, or -1 on error; call SDL_GetError() for more information.
+ *
+ * \since This function is available since SDL 3.0.0.
+ */
+extern SDL_DECLSPEC int SDLCALL SDL_ReadProcess(SDL_Process *process, void *buffer, int size);
+
+/**
+ * Read from a process' stderr.
+ *
+ * The process must have been created with the SDL_EXEC_STDERR flag, else this function will fail.
+ *
+ * \param process The process to read data from.
+ * \param buffer A buffer that will contain the data read from the process.
+ * \param size The amount of data to read from the process.
+ *
+ * \returns The number of bytes read, or -1 on error; call SDL_GetError() for more information.
+ *
+ * \since This function is available since SDL 3.0.0.
+ */
+extern SDL_DECLSPEC int SDLCALL SDL_ReadErrProcess(SDL_Process *process, void *buffer, int size);
+
+/**
+ * Stop a process.
+ *
+ * \param process The process to stop.
+ * \param force Stop the process immediately, without giving it a chance to free its resources.
+ *
+ * \returns true on success, false on error; call SDL_GetError() for more information.
+ *
+ * \since This function is available since SDL 3.0.0.
+ */
+extern SDL_DECLSPEC SDL_bool SDLCALL SDL_KillProcess(SDL_Process *process, SDL_bool force);
+
+/**
+ * Wait for a process to finish.
+ *
+ * This function must be called prior to destroying (or losing) any process.
+ *
+ * Process that died but aren't waited for remain zombie processes that take resources on the system.
```

Comment:
"*might* remain zombie processes"? POSIX has the zombie process concept but I suspect not all platforms do.